lib/feature_management.rb

```diff
@@ -151,8 +151,4 @@ def self.idv_by_mail_only?
       outage_status.any_phone_vendor_outage? ||
       outage_status.phone_finder_outage?
   end
-
-  def self.idv_allow_selfie_check?
```

Comment:
This is still stubbed in a lot of tests, where I think we'd want to remove the stubs for the method that doesn't exist (surprised if RSpec doesn't already raise on that).

One example (of 22 I found):

https://github.com/18F/identity-idp/blob/2a86549cc06ce3151f78e871e21c9ea7add78cfc/spec/controllers/idv/hybrid_mobile/document_capture_controller_spec.rb#L79

Reply:
D'oh! Thanks for catching this.

Reply:
`allow()` doesn't assert if not called, but `expect()` will

Reply:
I guess more specifically I would have thought that RSpec mocks would raise when trying to "receive" something that doesn't exist on the object, but I suppose that might not be possible with metaprogramming features like `method_missing`.

Reply:
Oh, and some of those tests were specifically testing FF-off behavior so I need to do some careful removal of tests, vs. just ripping out the assignments.